lib/signaling/v2/twilioconnectiontransport.js

```diff
@@ -308,7 +311,7 @@ class TwilioConnectionTransport extends StateMachine {
       this.preempt('disconnected', null, [error]);
       this._sendConnectOrSyncOrDisconnectMessage();
       this._twilioConnection.close();
-      this._eventPublisher.disconnect();
+      this._disconnectEventPublisher();
```

Comment:
again, It does not feel right that _disconnectEventPublisher is declared as property. I am curious what makes it property v/s class method.

Reply:
@makarandp0 See my response [here](https://github.com/twilio/twilio-video.js/pull/1466#discussion_r632741895).